pkg/interceptors/imagechecker/interceptor.go

```diff
@@ -85,8 +86,17 @@ func (i *Interceptor) wait(ctx context.Context, image string) error {
 			if found {
 				return nil
 			}
+			if try == 0 {
+				fmt.Println("Deployment is waiting for image to be built.")
+				try++
+				continue
+			}
+			if (try % 4) == 0 {
+				fmt.Println("Still waiting for image to be available...")
+				try++
```

Comment:
You will never reach `try > 1`, because you are only incrementing inside of those two conditions.